src/locales/hu.json

```
@@ -25,7 +25,13 @@
 		},
 		"howto": {
 			"description": "Kiír utasításokat a gyakori parancsok használatához.",
-			"name": "hogyankell"
+			"name": "hogyankell",
+			"responses": {
+				"to-submit": "Dal beküldéséhez használja a {command} parancsot\n\nPéldául: {example}\nMegerősítéssel válaszolok, jelezve, hogy a dalod felkerült a sorba!",
```

"Megerősítéssel válaszolok, jelezve, hogy a dalod felkerült a sorba!"

This is a nice and exact translation of the English string, but in Hungarian I'd rather say "Jelezni fogom, amint sikeresen hozzáadtam a dalt a sorhoz!" which roughly equals "I will notify [you] once I've successfully added the song to the queue"

```suggestion
				"to-submit": "Dal beküldéséhez használja a {command} parancsot\n\nPéldául: {example}\nJelezni fogom, amint sikeresen hozzáadtam a dalt a sorhoz!",
```